dotnet/src/SemanticKernel.UnitTests/Utilities/AIConnectors/FunctionCallsProcessorTests.cs

```diff
@@ -0,0 +1,622 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+#pragma warning disable IDE0005 // Using directive is unnecessary
+using Microsoft.SemanticKernel.Connectors.AI;
+#pragma warning restore IDE0005 // Using directive is unnecessary
+using Xunit;
+
+namespace SemanticKernel.UnitTests.Utilities.AIConnectors;
+public class FunctionCallsProcessorTests
```

Comment:
What % code coverage are we getting?

Reply:
With a few new unit tests the coverage is 80%/85%. The only uncovered functionality is logging:
<img width="604" alt="image" src="https://github.com/user-attachments/assets/3fbc82d5-8094-42df-b158-84d2da5bf53e">